readme.md

```diff
@@ -464,6 +464,7 @@
 - [Lockpicking](https://github.com/meitar/awesome-lockpicking#readme) - The art of unlocking a lock by manipulating its components without the key.
 - [Cybersecurity Blue Team](https://github.com/meitar/awesome-cybersecurity-blueteam#readme) - Groups of individuals who identify security flaws in information technology systems.
 - [Fuzzing](https://github.com/cpuu/awesome-fuzzing#readme) - Automated software testing technique that involves feeding pseudo-randomly generated input data.
+- [GDPR](https://github.com/bakke92/awesome-gdpr#readme) - Privacy tools and resources.
```

Comment:
Maybe something closer to what's in the repo is a bit better as a description:

> GDPR-compliant tools for website creators

Alternatively the acronyms could be expanded as well, maybe?

> European Union Global Data Protection Regulation (GDPR) compliant tools for website creators

Reply:
I agree that using the words "tools" would be describing the content. It is not merely just for website creators though. And the usage of "tools and resources" to describe isn't according to the requirements for Awesome repos when describing it in the readme document.
As I understand it is it OK in the Awesome-list though.

Reply:
Ah, ok. Actually it makes sense. I now understand a bit better the requirements (upon re-reading the guidance).

Reply:
> As I understand it is it OK in the Awesome-list though.

No, it should explain the theme, in this case GDPR, not the list itself.